notifications_utils/version.py

```diff
@@ -1 +1 @@
-__version__ = '10.0.0'
+__version__ = '10.0.1'
```

Comment:
Take point about statsd. Nothing else should be affected - it doesn't change signatures?